taskcluster/ci/signing/kind.yml

```diff
@@ -28,6 +29,7 @@ job-template:
     description: Sign MozillaVPN
     worker: {} # Will be configured by signing:transforms
     run-on-tasks-for: []
+    push-to-release-index: true
```

Comment:
nit: I think `push` is probably the wrong verb here as indexes are just pointers to a task. Maybe `add-release-index`?

Note that if you go with the `index_builders` suggestion above, this key won't be necessary.